packages/react-native-renderer/src/ReactFabric.js

```diff
@@ -124,6 +125,10 @@ export {
   // This method allows it to acess the most recent shadow node for
   // the instance (it's only accessible through it).
   getNodeFromInternalInstanceHandle,
+  // Fabric native methods to traverse the host tree return the same internal
+  // instance handles we use to dispatch events. This provides a way to access
+  // the public instances we created from them (potentially created lazily).
+  getPublicInstanceFromInternalInstanceHandle,
```

Comment:
What's the difference between a Node and a PublicInstance (i.e. why doesn't the existing `getNodeFromInternalInstanceHandle` work)?

Reply:
Node is the shadow node (which is internal). Public instance is the instance of `ReactFabricHostComponent` that's provided via refs.

Reply:
I wonder if these are supposed to be different things? I think I would expect a Node to be a HostComponent but I don't have a ton of context.